apps/quickstart-react/package.json

```diff
@@ -15,8 +15,9 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
-    "monday-sdk-js": "^0.1.2",
-    "monday-ui-react-core": "0.0.48",
+    "kill-port": "^1.6.1",
+    "monday-sdk-js": "^0.1.3",
+    "monday-ui-react-core": "1.20.0",
```

Comment:
Note that you upgraded a major version, did we make sure that nothing is broken?

Reply:
Everything looked good (we did not did a very deep check if that is the information you need). Is there any specific flow that you want to verify?